builder/softlayer/step_capture_image.go

```diff
@@ -40,6 +86,33 @@ func (self *stepCaptureImage) Run(state multistep.StateBag) multistep.StepAction
 		return multistep.ActionHalt
 	}
 
+	if config.ImageType == "standard" {
```

Comment:
Oh, this works nicely :) 

I never would have guessed that the image would show up in the list before image creation is finished. I'll make the appropriate changes.